subcategory/subcategory.py

```diff
@@ -108,8 +110,8 @@ def generate_subcategories(generator, writer):
         dates = [article for article in generator.dates if article in articles]
         write(subcat.save_as, subcategory_template, generator.context, 
                 subcategory=subcat, articles=articles, dates=dates, 
-                paginated={'articles': articles, 'dates': dates},
```

Comment:
I'll still get error `articles_page undefined` in latest master and must add this line back in order to get paginated articles in the template.